play/src/front/Connexion/RoomConnection.ts

```diff
@@ -427,6 +428,15 @@ export class RoomConnection implements RoomConnection {
                             commandsToApply,
                         } as RoomJoinedMessageInterface,
                     });
+
+                    // Check WebRtc connection
+                    if (roomJoinedMessage.webrtcUserName && roomJoinedMessage.webrtcPassword) {
+                        checkCoturnServer({
+                            userId: this.userId,
+                            webRtcUser: roomJoinedMessage.webrtcUserName,
+                            webRtcPassword: roomJoinedMessage.webrtcPassword,
+                        });
+                    }
```

Comment:
Let's keep this code out of the RoomConnection.

Maybe create a `WebRtcTester` class that is instantiated in the GameScene and that listens to the `roomJoinedMessageStream` directly from the RoomConnection?

Reply:
Because I have all data like `webrtcUserName` and `webrtcPassword` here so can we apply the WebRtc check here?